.github/workflows/linter.yml

```diff
@@ -1,31 +1,71 @@
----
-name: Lint Code Base
+name: Lint
 
 on:
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
+
+permissions:
+  checks: write
+  contents: write
 
 jobs:
-  build:
-    name: Lint Code Base
+  lint-python:
+    name: Lint Python
     runs-on: ubuntu-latest
 
-    permissions:
-      contents: read
-      packages: read
-      statuses: write
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check changed py files
+        id: changed-files
+        uses: tj-actions/changed-files@v24
+        with:
+          files: |
+            **/*.py
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
+        with:
+          python-version: 3.8
+
+      - name: Install Python dependencies
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
+        run: pip install flake8
+
+      - name: Run flake8 linter
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
+        uses: wearerequired/lint-action@v2
+        with:
+          flake8: true
+          flake8_auto_fix: false
+          flake8_args: ${{ steps.changed-files.outputs.all_changed_files }}
+          continue_on_error: false
+
+  # If this is in the same step as python linting the annotations don't show up
+  lint-javascript:
```

Comment:
Can we drop this section entirely?  I don't believe commcare-cloud contains any javascript files

Reply:
You're right, I'll make the update.